packages/contracts/src/combat/LibActiveCombat.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import { ActiveCombat } from "../codegen/Tables.sol";
+
+library LibActiveCombat {
+  error LibActiveCombat__CombatActive();
+  error LibActiveCombat__CombatNotActive();
+  error LibActiveCombat__CombatActiveForDifferentRetaliator();
+
+  function getRetaliatorEntity(bytes32 initiatorEntity) internal view returns (bytes32 retaliatorEntity) {
+    if (ActiveCombat.get(initiatorEntity) != bytes32(0)) {
```

Comment:
By this condition the only valid entity would be 0, which is the opposite of what we want